.gitignore

```diff
@@ -142,6 +142,8 @@ $tf/
 _ReSharper*/
 *.[Rr]e[Ss]harper
 *.DotSettings.user
+.idea/
```

Comment:
Tried using JetBrains Rider with CLI on Mac. Adding ignores for their artifacts.